go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1473,6 +1474,11 @@ func flowsForDefaultBridge(bridge *bridgeConfiguration, extraIPs []net.IP) ([]st
 					"actions=set_field:%s->eth_dst,output:%s",
 					defaultOpenFlowCookie, netConfig.v4MasqIPs.ManagementPort.IP,
 					bridgeMacAddress, netConfig.ofPortPatch))
+			dftFlows = append(dftFlows,
```

Comment:
That's correct, This means we also want to marking on sgw.
From looking at the code we are already doing the iptables DNATs etc in SGW too so it would make sense to keep that behavior consistent.